.github/workflows/release.yml

```diff
@@ -131,11 +109,9 @@ jobs:
         with:
           signing_key_file: ${{ secrets.SIGNING_KEY_FILE }}
           signing_file_path: ${{ env.SIGNING_KEY_FILE_PATH }}
-      - name: Set github user
```

Comment:
Remind me why this is being removed? We are creating a shared GitHub user for releases right?